.eslintrc

```diff
@@ -0,0 +1,24 @@
+{
+    "extends": [
+      "react-app",
```

Comment:
rename your PR title to some meaningful name.

Reply:
noted